docs/admin/kubeadm-alpha.md

```diff
@@ -0,0 +1,266 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm alpha
+notitle: true
+---
+
+# Kubeadm alpha
+`kubeadm alpha` provides a preview of a set of features made available for gathering feedback
+ from the community. Please try it out and give us feedback!
+{: .caution}
+
+In v1.8.0, kubeadm introduced `kubeadm alpha phase` command with the aim to make kubeadm more 
+modular, letting you invoke atomic sub-steps of the bootstrap process; basically, you can 
```

Comment:
Suggestion:
In v1.8.0, kubeadm introduced the `kubeadm alpha phase` command with the aim of making kubeadm more modular. This modularity enables you to invoke atomic sub-steps of the boostrap process; you can let kubeadm do some parts and fill in yourself where you need customizations.